torch/csrc/jit/runtime/register_cuda_ops.cpp

```diff
@@ -36,14 +36,14 @@ RegisterOperators const reg({
         },
         aliasAnalysisFromSchema()),
     Operator(
-        "cuda::_current_device() -> int",
+        "aten::current_device() -> int",
         [](Stack* stack) {
           auto v = c10::cuda::current_device();
           push(stack, static_cast<int>(v));
         },
         aliasAnalysisFromSchema()),
     Operator(
-        "cuda::_set_device(int64_t val) -> ()",
+        "cuda::_setDevice(int64_t val) -> ()",
```

Comment:
We mostly don't use camel case for functions so I'm not sure why the casing got changed here.

Reply:
This was mainly done to have the JIT API's as close to eager API's as possible. `torch._C._cuda_setDevice` -> `torch.cuda._setDevice`. That was the only motivation. I can change it to `_set_device`